server/network.go

```diff
@@ -111,6 +113,18 @@ func newLocalNetwork(opts localNetworkOptions) (*localNetwork, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	var defaultConfig, globalConfig map[string]interface{}
```

Comment:
probably the names defaultConfig, globalConfig are misleading (globalConfig is named defaultConfig elsewhere), shall try to preserv defaultConfig related naming for the last one.
suggestion... staticDefaultNodeConfig, userProvidedDefaultNodeConfig, customNodeConfig. Anyway, just a suggestion.

Reply:
Applied `globalConfig` to be consistent everywhere